### PR TITLE
Add dependancy context to recorder

### PR DESCRIPTION
**Pull request by matthiasdebaat**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/recorder.markdown [hunk 1]
```diff
@@ -12,7 +12,7 @@ ha_codeowners:
 ha_integration_type: system
 ---
 
-The `recorder` integration is responsible for storing details in a database, which then are handled by the [`history`](/integrations/history/) integration.
+The `recorder` integration is responsible for storing details in a database, which then are handled by the [`history`](/integrations/history/) integration. This integration is by default enabled as dependency of the [`history`](/integrations/history/) integration.
```
Comment (balloob) [inline source/_integrations/recorder.markdown]: I don't think that we need to link history twice. Maybe just mention "This integration is enabled by default."

Comment (frenck) [reply source/_integrations/recorder.markdown]: I think the whole relation with the history integration can be removed from this part.

Comment (matthiasdebaat) [reply source/_integrations/recorder.markdown]: > I think the whole relation with the history integration can be removed from this part.

I agree, removed it.

**Review by frenck — APPROVED** [review]
Thanks, @matthiasdebaat 👍 

../Frenck

**Pull request merged**